ee/api/backup.py

```diff
@@ -0,0 +1,51 @@
+import json
```

Comment:
looks like this causes the FOSS repo to run into an error 
```
ModuleNotFoundError: No module named 'ee'
```
so I'm not sure where it should be in the end

Reply:
Yes because in FOSS ee/ folder is removed. Your import there needs to be conditional on whether ee is enabled.